Jenkinsfile

```diff
@@ -162,7 +162,7 @@ pipeline {
                   unstashV2(name: 'build', bucket: "${JOB_GCS_BUCKET_STASH}", credentialsId: "${JOB_GCS_CREDENTIALS}")
                   dir("${BASE_DIR}") {
                     withOtelEnv() {
-                      sh label: 'mvn test', script: './mvnw test'
+                      sh label: 'mvn test', script: './mvnw test -DargLine=\"-Delastic.apm.overwrite.config.docs=false\"'
```

Comment:
I'm assuming you intentionally sneaked in the changes to fix our documentation tests here? I'm fine with that, just asking to make sure it was intentional.

Depending on which PR gets merged first, we need to make sure to also add this to GH actions (#2870).